pyproject.toml

```diff
@@ -231,6 +231,9 @@ dev = [
   # https://opendev.org/openstack/reno/src/branch/master/requirements.txt#L7
   "dulwich>=0.21.0,<1.0.0",
 ]
+gradient = [
+  "gradientai==1.1.0"
+]
```

Comment:
For Haystack 2.0 we're abandoning this approach. Let's remove it, `LazyImport` takes care of the error messages already :)

Reply:
done!

Reply:
done!